wezterm-gui/src/customglyph.rs

```diff
@@ -4804,6 +5081,7 @@ impl GlyphCache {
                     }],
                     &mut buffer,
                     PolyAA::AntiAlias,
+                    None,
```

Comment:
this sort of `None` parameter makes it hard to intuit the purpose; how about using `BlendMode::default()` here instead of passing an `Option`?

Reply:
Sure, works for me.